app/src/components/Dashboard/ChartTabs.tsx

```diff
@@ -1,4 +1,5 @@
 import { Tab, TabList, TabPanel, TabPanels, Tabs } from '@chakra-ui/react';
+import { TextsOverTimeChart } from 'components/Graphs/TextsOverTimeChart';
```

Comment:
Just to save you some pain, this is what is causing the workflows to fail. It's not happy with absolute imports sadly